## Background


:::: {.columns}

::: {.column width="45%"}

![Circulating cell free DNA^[Adapted from Wikipedia]](assets/fig/CtDNA_in_circulation.png)

:::

::: {.column width="55%"}


- **Circulating free DNA** (cfDNA) are DNA fragments released into bloodstream 

- Fraction of cfDNA could be released from cancer or tumor cells are **circulating-tumor DNA** (ctDNA)

- Contains genetic and epigenetic changes, and could reveal the cells from which is originated
  - Identify different types of cancer


:::

::::

::: {.notes}

- ctDNA are like fraction of cfDNA but derived from tumor cells

- Apoptosis and necrosis are main sources of these ctDNA, these are simply two forms of cell death that naturally occurs in the body

:::

## Detecting the origin of cfDNA

Current cfDNA screening test can detect presence of abnormal signals but [cannot]{.quaternary} tell tumor's origin or cancer type or tissue of origin (**TOO**)

. . .

- Computation methods use epigenetic markers like [**methylation profiles**]{.primary} to deduce origin of cfDNA fragments
  - "Deconvolute" plasma cfDNA composition
  - Varying approach, probabilistic, linear model, matrix factorization, etc. 

. . .

::: {.callout-note}

## Existing methods limiations

- Cannot deconvolute multiple cancer tissues
- Do not account for missing variables due to incompleteness of atlas
- Do not allow full deconvolution of all cfDNA components and estimate cell proportion only

:::

::: {.notes}

- Explain why using these epigentic markers, and why each is important

- Reference atlas consist of either normal tissues or celltype specific methylation marker

- Models could be linear combinations or multimodal analysis or support vector machine, some of them introduce feature selection to remove redundant markers, others use probabilitis model. 

:::


## MetDecode

Authors^[@passemiers2024metdecode] came up with their reference-based deconvolution method

. . .

In some sense, ["combined"]{.secondary} existing methodology like nonnegative least squares, matrix factorization etc.

- Also built a newer reference atlas of tissue-specific methylation markers for 4 different cancer tissues
  - Breast, ovarian, cervical and colorectal

- With option to **extend reference atlas** with unknown methylation patterns on-the-fly

---

![](assets/new_fig.jpg){.nostretch fig-align="center" width=100%}


::: {.notes}

- Constructed DNA methylation atlas for multiple blood cell types and tumor tissues from public databases BLUEPRINT, TCGA and their in house samples

- DMRs were determined by extending CpG sites with >30% differential methylation to regions of at least 4 CpGs.

- Then cpg count matrices from these DMRs would be used in their deconvolution algorithm to estimate cell type contributions to cfDNA

:::



## The main deconvolution algorithm

$$
f(A) \quad = \quad \sum\limits_{i=1}^n \sum\limits_{k=1}^p \quad W_{ik} \quad \Big| \underbrace{R_{ik}^{\text{(cfdna)}}}_{(1)} - \underbrace{\sum\limits_{j=1}^m A_{ij} B_{jk}}_{(2)}\Big| 
$$

1. Methylation ratios $R^{\text{cfdna}}$
2. Reconstructed matrix, which approximates $(1)$


::: {.callout-note collapse="true"}

## Some math behind how MetDecode address unknown cell type contributor

To account unknown contributors in cfDNA mixture by adding $h$ extra rows to $R^{\text{(atlas)}}$

$$
R_{hk}^{\text{(atlas)}} = \begin{cases} R_k^{lb}, \quad e_k > 0 \\ R_k^{ub}, \quad otherwise \end{cases} \quad \text{where} \quad e_k = \text{median}_i \quad \Big( -R_{ik}^{(cfdna)} + \sum\limits_{j} \alpha_{ij} R_{jk}^{(\text{atlas})} \Big)
$$

:::


::: {.notes}

- The unknown contributing part is actually to the atlas
- $e_k > 0$ implies reconstructed cfDNA samples have higher methylation ratios than observed in ùëÖ(cfdna), suggesting missing information in R (atlas)
- lb is lower bound of quantile (0.4, Rk(atlas)), ub is upper bound of quantile (0.6 Rk(atlas))
- These left bound and right bound are calculated by quantiles 
- Alpha represents initial cell type proportion estimates, obtained from non-negative least squares
- $A_{ij}$ is estimated proportion of cell type $j$ to cfDNA profile $i$, found by minimizing weighted mean absolute error
- k is a marker region, $R_k$ is k-th column of R(atlas)

:::

::::



## Evaluation metrics

- Pearson Correlation Coefficient $\rho$ and Mean Squared Error (MSE) to evaluate MetDecode estimations

- Accuracy to evaluate multiclass cancer TOO prediction, and Cohen's kappa to adjust for multiclass nature of the problem

. . .

::: {.callout-note}

## Some notations

$$
MSE = \quad \frac{1}{n} \sum\limits_{i=1}^{n} \quad (Y_i - \hat{Y_i})^2 
$$

$$
\begin{align*}
\kappa &= \frac{(p_o - p_e)}{(1 - p_e)}, \quad p_e = \frac{1}{N^2} \sum\limits_{k=1}^K n_{k1} n_{k2}
\end{align*}
$$

where $n_{k1}$ is number of times label $k$ appears in predictions, and $n_{k2}$ is  number of times label $k$ is a true label^[@artstein2008inter]

:::



## Creation of reference atlas and marker selection {.smaller}

:::: {.columns}

::: {.column width="50%"}

![Heatmap displaying the methylation ratios of all the selected marker regions](assets/fig/fig1_heatmap.png)

:::

::: {.column width="45%"}

<br>
<br>

- READ had greatest number of siginifcant DMRs
  - In contrast OV and CESC had lowest, due to lower coverage of these samples

::: {.fragment}

- [Three variants]{.quaternary} of deconvolution setting came up to address above:
  1. Using all marker regions
  2. Only signifcant marker regions
  3. 23 [most discriminative]{.secondary} marker regions for each cell type, whereas 23 is roughly minimal number of DMR found for a cell type

:::


:::

::::

::: {.notes}

- Rectum adenocarcinoma (READ)
- Ovarian carcinoma (OV)
- cervical squamous cell carcinoma (CESC)
- discriminative means sorted by P-value

- colour scale from hypermethylation to hypomethylation.


:::


## Coverage-based weighthing in MetDecode {.smaller}

:::: {.columns}

::: {.column width="55%"}

![Evaluation of the coverage-based weighting used in MetDecode](assets/fig/fig3_coverage.png)

:::

::: {.column width="45%"}

<br>

- Ran on 50 simulation runs, each containing $5000$ simulated cfDNA samples.

- Then computed Pearson Correlation Coefficient of different deconvolution algorithms

- Upon averaging all correlation coefficients, MetDecode was [significantly higher]{.quaternary} than all other approaches
  - [BUT]{.secondary} not for looking at blood cell types only

:::

::::

::: {.notes}

- Extra noise could come from sequencing tech difference, and mappability of genomes
- So, they reweighted their objective to lower contribution of lower coverage marker regions, since higher coverage enables more accurate estimate

- For each cell type, one-sided T-test to assess difference of MetDecode and best of four others (including MetDecode without coverage)

- Simulations designed based on real data with random noise, for computation speed, used shortlisted 23x13 DMRs only and compare them with other algorithms

- But performance much higher on cancer components, and not in blood cell types

- The parenthesis one were p-value , all ran as one-sided t-test

:::


## Accuracy of complex mixture deconvolution {.smaller}


![Deconvolution of genomic DNA methylation profiles](assets/fig/fig6_full.png)

- [High]{.primary} correlation when compraing complete blood counting and MetDecode deconvolution estimates

- MetDecode with no unknown contributor outperformed NNLS in terms of average pearson correlation and MSE
  - Adding unknown contributor should've [increased]{.quaternary} performance though, but did not

::: {.notes}

- Overall, these results demonstrate that MetDecode can accurately estimate proportions in samples containing a mixture of cell types. 

:::



## Identify TOO in cfDNA from cancer patients

![Cancer type prediction comparisons based on highest cancer contributors](assets/fig/cancer_pred-highlight.png)


- MetDecode with 1 unknown contributor performs best based on Cohen'kappa

- All methods do [equally poor]{.secondary} for $< 50\%$ accuracy when predicting all samples

- Closer performance when looking at those $19$ samples with tumor fraction $> 3\%$^[This is based on ichorCNA from @adalsteinsson2017scalable]
  - This is its $84.2\%$ accuracy of correct TOO in $16/19$ cancer cases


::: {.notes}

- Highest cancer contributor here has special meaning, given its multicass, author convert the way to see if prediction is normal or one of the TOO. This TOO is determined by taking some normalization, whereas highest is considered the TOO

- So when method estimated all cancer contributor as zero, the predicted class is considered normal, otherwise it is the highest normalized contributor

-  Half of the misclassified samples were classified as ovarian cancers

- These samples were run with another tool ichorCNA which estimates tumor fraction in cell free DNA

- One mucinous ovarian carcinoma (case 45, stage IV) was predicted to have breast cancer tissue as a major cancer contributor. One triple-negative breast cancer sample (case 8, stage I) was misclassified as ovarian.

- Bottom is ground truth

:::

## Conclusion


::: {.callout-tip}

## How could one utilize cfDNA?

cfDNA epigenetic signatures can be used to deduce TOO or cancer type

:::

. . .

MetDecode is an algorithm that estimates contributions and type of cancer in cfDNA sample


- It models unknown contributors not present in the reference atlas

- And accounts for coverage of each marker region to alleviate potential sources of noise


## Limitations and Future Direction

- [**Limited size**]{.primary} of cfDNA samples for different cancer types
  - Total 93 samples, 4 being Cervical, 13 being Ovarian, rest are breast and colorectal


- Deconvoluting and defining the TOO will aid the oncologists in identifying the tumor and direct treatment
  - Specially when invasive examinations and radiological investigation are not ideal



## Some comments

- Why weighting approach only improves deconvolution accuracy on cancer components only and not in blood cell types?

- Why sometimes adding extra unknown contributor yields better result and sometimes not?

- Cell type deconvolution still seems hard (low accuracy in terms of predicting cancer type), what is the next step?

- ~~[Aside]{.secondary}, can you always just combined existing approach to get a "new" method out?~~


# Thanks! {.thanks}


## Reference



